src/AdminPage.jsx

```diff
@@ -0,0 +1,232 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Table from '@mui/material/Table';
+import TableBody from '@mui/material/TableBody';
+import TableCell from '@mui/material/TableCell';
+import TableContainer from '@mui/material/TableContainer';
+import TablePagination from '@mui/material/TablePagination';
+import TableRow from '@mui/material/TableRow';
+import Typography from '@mui/material/Typography';
+import Paper from '@mui/material/Paper';
+import axios from 'axios';
+import NavBar from './components/NavBar.jsx';
+import Footer from './components/Footer.jsx';
+import DeleteIcon from '@mui/icons-material/Delete';
+import { useNavigate } from 'react-router-dom';
+import EnhancedTableHead from './components/AdminHeader.jsx';
+
+const getCars = async () => {
+  const { data } = await axios.get('http://localhost:3000/cars');
+  return data;
+}
+
+let rows = await getCars();
+
+const changeAvailability = async (id) => {
```

Comment:
so for all of the admin calls, you'll want to ensure that it's authenticated. We want to verify from the backend that the person doing this call is an admin because hypothetically any user could manipulate the front end and load this page (or just call that URL themselves directly)